README.md

```diff
@@ -63,9 +63,9 @@ but not limited to:
 
 
 ## GETTING STARTED
-* You want to start the RIOT? Just follow our [quickstart guide](http://doc.riot-os.org/index.html#the-quickest-start) or the [getting started documentation](http://doc.riot-os.org/getting-started.html).
+* You want to start the RIOT? Just follow our [quickstart guide](http://doc.riot-os.org/index.html#the-quickest-start) or try this [tutorial](https://github.com/RIOT-OS/Tutorials/blob/master/README.md). For specific toolchain installation, follow these [instructions](http://doc.riot-os.org/getting-started.html).
```

Comment:
I would prefer to keep 'getting started instructions', which is more inviting and the page also mentions how to build an example at the end.

Reply:
mmh, well they are still pointed to (last link)?

Reply:
Anyways: the point is that it would be better to have a link to the nice tutorial we have from this README directly.

Reply:
So what do you propose more concretely ;)

Reply:
I think it's just about the text of the link (a user usualy only sees where it goes to when they hover the link) and I agree with @cladmi that it should at least contain "getting started" somewhere.

Reply:
Yeah my remark was on the text for the link.